bin/lib/release.js

```diff
@@ -56,6 +57,10 @@ export default (releaseType) => {
         throw new Error('No .gitignore file');
       }
 
+      let outputFile = path.resolve(__dirname, `../../styles/${OUTPUT_CSS_FILENAME}`);
+      console.log(`- Deleting output file ${outputFile}`.green);
+      yield pUnlink(outputFile);
```

Comment:
What happens if this file does note exist? DO we get an error? If so, you can use `fs.existsSync` to check it.